client/actions/assignments/api.ts

```diff
@@ -393,10 +394,11 @@ const revert = (item) => (
 
 /**
  * Action to lock an assignment
- * @param {String} item - Assignment to be unlocked
+ * @param {IAssignmentItem} assignment - Assignment to be unlocked
+ * @param {String} action - The action to assign to the lock
```

Comment:
I think we don't need them anymore. Everything is clear with types itself.

Reply:
Agree'd, just habit I suppose.